lemur/tests/conf.py

```diff
@@ -34,6 +34,7 @@ def get_random_secret(length):
 
 # You should consider storing these separately from your config (should be URL-safe)
 LEMUR_TOKEN_SECRET = "test"
+LEMUR_TOKEN_SECRETS = [LEMUR_TOKEN_SECRET]
```

Comment:
For these files where we control the config, is there any reason to keep `LEMUR_TOKEN_SECRET`? No need for backwards compatibility here.